### PR TITLE
FSET-1725 Resetting candidate removal reason

### PR DESCRIPTION
**Pull request by vglushak**

### CODE REVIEW

File: app/controllers/CandidateAllocationController.scala [hunk 1]
```diff
@@ -95,4 +95,11 @@ trait CandidateAllocationController extends BaseController {
     }
   }
 
+  def resetEventsRemovals(applicationId: String) = Action.async { implicit request =>
```
Comment (kjharkin) [inline app/controllers/CandidateAllocationController.scala]: Name of method doesn't really reveal it's intention.

File: app/controllers/CandidateAllocationController.scala [hunk 2]
```diff
@@ -95,4 +95,11 @@ trait CandidateAllocationController extends BaseController {
...
+    candidateAllocationService.removeEventsRemovals(applicationId).map { _ =>
+      Ok("")
```
Comment (kjharkin) [inline app/controllers/CandidateAllocationController.scala]: Should return ```NoContent```

File: app/repositories/CandidateAllocationRepository.scala [hunk 3]
```diff
@@ -32,11 +32,12 @@ import scala.concurrent.Future
 
 trait CandidateAllocationRepository {
   def save(allocations: Seq[CandidateAllocation]): Future[Unit]
-  def findNoShowAllocations(applications: Seq[String]): Future[Seq[CandidateAllocation]]
   def findAllAllocations(applications: Seq[String]): Future[Seq[CandidateAllocation]]
   def activeAllocationsForSession(eventId: String, sessionId: String): Future[Seq[CandidateAllocation]]
   def allocationsForApplication(applicationId: String): Future[Seq[CandidateAllocation]]
   def removeCandidateAllocation(allocation: CandidateAllocation): Future[Unit]
+  def removeEventsRemovals(applicationId: String): Future[Unit]
```
Comment (kjharkin) [inline app/repositories/CandidateAllocationRepository.scala]: name again

File: app/repositories/CandidateAllocationRepository.scala [hunk 4]
```diff
@@ -122,6 +117,14 @@ class CandidateAllocationMongoRepository(implicit mongo: () => DB)
     collection.update(query, update) map validator
   }
 
+  def removeEventsRemovals(applicationId: String): Future[Unit] = {
```
Comment (kjharkin) [inline app/repositories/CandidateAllocationRepository.scala]: name

**Comment by vglushak** [comment]
All comments addressed.

**Pull request merged**